openmc/weight_windows.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
+
 from collections.abc import Iterable
 from numbers import Real, Integral
```

Comment:
Our Python code follows [PEP8](https://peps.python.org/pep-0008/), which recommends grouping imports into three groups: standard-library modules, third-party modules, and local imports. In your case, both `__future__` and `typing` are standard-library and should be in the first block:
```suggestion
from __future__ import annotations
from collections.abc import Iterable
from numbers import Real, Integral
from typing import Iterable, List
```